### ISSUE TITLE
C# API support for .NET over Windows

### ISSUE THREAD

**Issue by eranroll** [eclipse-cyclonedds/cyclonedds#1176]
Hi,

Is there support for using Cyclone DDS, running on Windows, with bindings for C# applications?

Regards,

**Comment by eboasson** [comment]
No, sadly we don't have a C#/.NET binding yet. We do want one but time is limited, so if you need it today you may have to do it yourself ... A minimal C# binding isn't that much work and we'd be happy to assist with answering any questions you might have.

**Comment by eranroll** [comment]
Good to here! Thank you very much, I will keep following 🙏🏻

**Comment by jspg** [comment]
Hi,

Are there any updates on this?

Best regards

**Comment by NoxPardalis** [comment]
Hey @jspg, thanks for showing interest in the C# API :+1:.

It's is still being actively worked on, but It was moved to a side project for a while due to other work that needed to be prioritized (you know how it goes :sweat_smile:).

That work has simmered down somewhat, so I'm back to working on the C# API in earnest. There are two or three issues that still need to be resolved, and the IDLC back-end still needs to be finalized, but otherwise it's converging towards an MVP that the community can have fun testing.
